cypress/pages/checkoutInformationPage.js

```diff
@@ -1,28 +1,33 @@
-import { returnObjectSearchingByKey } from "../support/utils/fixtureInteraction";
+import findObjectByKey from '../support/utils/fixtureInteraction';
```

Comment:
Javascript doesn't care if you use a single quotation or double quotation, both are available on JS, also doesn't care about the format or type of import, you only need the correct name of the method, class, or variable pointing to the proper path of the root.
Also, both kinds of imports are available, name Import and default Import

<img width="561" alt="Screenshot 2024-08-02 at 9 56 48 AM" src="https://github.com/user-attachments/assets/89fc25d4-fc5e-4cc4-9721-84a26ba530e5">
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import